src/main/java/uk/gov/companieshouse/transactions/web/transformer/confirmation/impl/ConfirmationTransformerImpl.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.companieshouse.transactions.web.transformer.confirmation.impl;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import org.omg.CORBA.PRIVATE_MEMBER;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.transactions.web.confirmation.ConfirmationDescription;
+import uk.gov.companieshouse.transactions.web.exception.UnsupportedTransactionConfirmationException;
+import uk.gov.companieshouse.transactions.web.model.confirmation.Confirmation;
+import uk.gov.companieshouse.transactions.web.transformer.confirmation.ConfirmationTransformer;
+
+@Component
+public class ConfirmationTransformerImpl implements ConfirmationTransformer {
+
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("d MMMM yyyy");
+
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("h:mm a");
+
+    private static final String EMAIL_KEY = "email";
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Confirmation getTransactionConfirmation(Transaction transaction) throws UnsupportedTransactionConfirmationException {
+
+        Confirmation confirmation = new Confirmation();
+
+        confirmation.setCompanyName(transaction.getCompanyName());
```

Comment:
I know whitespace is a preference but I don't think its really useful between `setters`, could we remove the lines, its very hard to read here and it took me to scan each line to see what we are doing.

Reply:
Removed some whitespacing so hope that's more legible?

Reply:
That's better :+1:

Reply:
Ideal